input/templates/partials/myaccount/my-account-sidebar.hbs

```diff
@@ -20,6 +20,9 @@
     </a>
   </div>
   <div {{#if positionSeven}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items {{#if positionSeven}}active{{/if}}">
+    <a href="{{content.sidebarItemSeven.url}}">{{i18n "my-account:myAccountSidebar.changePassword"}}</a>
```

Comment:
Ah, another thing that it was wrong before. But I fixed it in my own version.
To avoid merging problems with my version, could you just replace the version here with this one: https://github.com/commercetools/commercetools-sunrise-java/blob/master/conf/templates/myaccount/my-account-sidebar.hbs
And then add the new tab with a URL structure similar to the one I show there? It will be faster this way.